### ISSUE TITLE
Możliwość uruchomienia programu w przeglądarce

### ISSUE THREAD

**Issue by Nircek** [Szymon300101/H3Xes#1]
Nie wiem czy spotkałeś się ze środowiskiem `p5.js`, ale jest to ogólnie biblioteka dla JavaScriptu bazująca na Processingu. Po przepisaniu tego kodu na `p5.js` i podpięciu pod repozytorium GitHub Pages, mógłbyś mieć stronę postaci `Szymon300101.github.io/H3Xes`, na której możnaby generować kolejne grafiki. Myślę, że jest to ciekawy pomysł nauczenia się czegoś nowego, a przy okazji jakbyś chciał się pochwalić tym projektem to wystarczy, że wyślesz komuś linka i sam jest w stanie poczuć moc proceduralnego generowania. Jakbyś chciał jakiś przykład to mogę odesłać do [mojego projektu](https://github.com/Nircek/hello-processing). Jeżeli nie spotkałeś się wcześniej z JSem to żeby przepisać ten kod na JSa wystarczy, że poznasz jakieś podstawowe zasady składni.

**Comment by Nircek** [comment]
W sumie to jeżeli byłbyś zainteresowany to mógłbym Ci pomóc i nauczyć czegoś. :stuck_out_tongue:

**Comment by Szymon300101** [comment]
O p5.js słyszałem, ale cały czas nie miałem czasu się nim zająć.
Odstraszało mnie to że to jest jednak inny język, a jeszcze nigdy nie
pisałem nic w języku niekompilowanym. Ale rzeczywiście, teraz jak piszesz,
to myślę że to nie jest głupi pomysł. Możliwe że nawet jeszcze dzisiaj się
tym pobawię. Natrafiłem wcześniej na poradniki od The Coding Train a, więc
myślę że z nich skorzystam. Są wspaniałe, i jeśli nie widziałeś, to
koniecznie zobacz, nawet jak już znasz środowisko.
Dzięki bardzo za wiadomość i zaangażowanie, jak coś w tym kierunku zrobię,
to Ci napiszę żebyś sobie zerknął.
Jak w ogóle mnie namierzyłeś, jesteś pierwszą osobą która mnie tu znalazła?
Pozdrawiam serdecznie,
Szymon.

niedz., 17 lis 2019, 09:01 użytkownik Marcin Zepp <notifications@github.com>
napisał:

**Comment by Nircek** [comment]
Prawdopodbnie chodziliśmy razem do gimnazjum (Katolik), aczkolwiek nigdy do Ciebie nie podbiłem, także pewien nie jestem. Kiedyś szukałem ludzi, z którymi można zrobić jakąś współpracę i znalazłem też np @jankulik. Processing znam tylko dzięki The Coding Train, także słyszałem. Jakbyś chciał pogadać to znajdziesz mnie np na fb. Btw wymieniony wyżej projekt to jest mój szczyt umiejętności w Processingu także jakoś genialny nie jestem. :smile:

**Comment by Szymon300101** [comment]
Żeczywiście skąś cię kojarzę. Ty teraz chodzisz do pika nie? Bo ja też. To
chyba wszyscy znają processinga od Shiffmana. Ten świat jest jednak mały.
No nic, jak coś zrobię to ci napiszę.
Do następnego.

niedz., 17 lis 2019, 15:15 użytkownik Marcin Zepp <notifications@github.com>
napisał:

**Comment by Nircek** [comment]
Tak, chodzę do pika. Świetnie Ci stronka wyszła, gratuluję. Możesz jeszcze ją podlinkować w opisie repozytorium, bo tak to trza ją ręcznie wpisywać.

**Comment by Szymon300101** [comment]
Pracuje nad tym, muszę Jeszce parę zeczy poczyścić.

pon., 18 lis 2019, 19:40 użytkownik Marcin Zepp <notifications@github.com>
napisał:
